core/src/test/scala/unit/kafka/tools/MirrorMakerTest.scala

```diff
@@ -17,12 +17,82 @@
 
 package kafka.tools
 
-import kafka.consumer.BaseConsumerRecord
+import java.util
+import java.util.Properties
+
+import kafka.consumer.{BaseConsumerRecord}
+import kafka.server.{KafkaConfig, KafkaServer}
+import kafka.tools.MirrorMaker.MirrorMakerNewConsumer
+import kafka.utils.{CoreUtils, TestUtils}
+import kafka.zk.ZooKeeperTestHarness
+import org.apache.kafka.clients.consumer.{ConsumerConfig, KafkaConsumer}
+import org.apache.kafka.clients.producer.{KafkaProducer, ProducerRecord}
 import org.apache.kafka.common.record.{Record, TimestampType}
 import org.junit.Assert._
-import org.junit.Test
+import org.junit.{After, Before, Test}
+
+class MirrorMakerTest extends ZooKeeperTestHarness {
```

Comment:
I suggest creating a new test under `integration/kafka/tools/MirrorMaker.scala` instead of reusing the existing unit test to avoid making the existing unit tests slower due to initialisation that they don't need.

Also, it seems like it may be better to inherit from `KafkaServerTestHarness` than `ZooKeeperTestHarness`?